src/main/utils/utils.ts

```diff
@@ -19,6 +19,10 @@ export class Utils {
         await vscode.commands.executeCommand('workbench.action.openSettings', `@ext:${Utils.getExtensionId()}` + (id ? ` ${id}` : ''));
     }
 
+    public static async openFeedback(): Promise<void> {
+        await vscode.env.openExternal(vscode.Uri.parse("https://github.com/jfrog/jfrog-vscode-extension/discussions/new?category=product-feedback"));
```

Comment:
I think that we will benefit more by letting the user to select the category
```suggestion
        await vscode.env.openExternal(vscode.Uri.parse("https://github.com/jfrog/jfrog-vscode-extension/discussions/new/choose"));
```